public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -475,6 +475,16 @@ export function buildGridItemForLibPanel(panel: PanelModel) {
   });
 }
 
+const hoverHeaderOffsetBehavior = (grid: DashboardGridItem) => {
+  grid.subscribeToState(() => {
```

Comment:
you are creating a subscription here, that you never unsubscribe from. This needs to beccome :
```ts
const sub = grid.subscribeToState(() => {
....
```

and then your behavior needs to return a deactivation handler that will unsubscribe:

```ts
return () => {
  sub.unsubscribe();
}
```

Reply:
@dprokop  a bit weird but I don't see when  `sub.unsubscribe()` is fired. For example `console.log('unsubscribe')` never shows up. Did I miss something?
```typescript
return () => {
  console.log('unsubscribe');  
  sub.unsubscribe();
}
```

Reply:
it's going to be called when you remove a panel or navigate away from a dashboard or go to panel edit. Literally when scene object get's deactivated (https://grafana.com/developers/scenes/advanced-activation-handlers)